src/patternfly/components/Page/page.scss

```diff
@@ -272,50 +279,51 @@
   margin-bottom: var(--pf-c-page__header-tools--MarginBottom);
   margin-left: auto; // to push it to the right
 
-  .pf-m-icons {
+  // Hides on mobile visible on desktop
+  .pf-m-desktop {
     display: none;
     visibility: hidden;
 
     @media screen and (min-width: $pf-global--breakpoint--lg) {
-      display: block;
+      display: inline-block;
```

Comment:
I think this should have been `display: flex;` since it applies to `.pf-c-page__header-tools-group`, which is `display: flex;`. But if you add `.pf-m-icons` back, just make sure whatever you use here works with that class and what you're using to show the desktop items.